app/views/admin/users/index.html.slim

```diff
@@ -24,7 +24,7 @@ table.users-table
         td
           = link_to user.email, impersonate_admin_user_path(user)
         td
-          = user.roles(&:to_s).join(", ")
+          = user.roles.map(&:to_s).join(', ')
```

Comment:
Removing your custom attr_accessor and subsequent getter and setter methods required this change since the roles aren't strings by default.